documentation/modules/proc-creating-kafka-cluster.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-kafka-cluster-{context}']
+= Creating a cluster
+
+Create a Kafka cluster, then a topic within the cluster.
+
+When you create a cluster, the Cluster Operator you deployed watches for new Kafka resources.
+
+.Prerequisites
+
+* For the Kafka cluster, a Cluster Operator is deployed
+* For the topic, a running Kafka cluster
+
+.Procedure
+
+. Log in to the Kubernetes cluster as a non-privileged user.
+
+. Create a new `my-cluster` Kafka cluster with 1 Zookeeper and 1 broker nodes.
```

Comment:
Shouldn't we use the word 'one' not the numeral 1? Also I think the word 'node' is potentially confusing. It has a different meaning in Kubernetes than in ZK. 

```suggestion
. Create a new `my-cluster` Kafka cluster with one Zookeeper and one Kafka broker.
```